website/aws.erb

```diff
@@ -1861,6 +1861,10 @@
                             <a href="/docs/providers/aws/r/s3_bucket.html">aws_s3_bucket</a>
                         </li>
 
+                        <li<%= sidebar_current("docs-aws-resource-s3-bucket-inventory") %>>
+                            <a href="/docs/providers/aws/r/s3_bucket_metric.html">aws_s3_bucket_inventory</a>
```

Comment:
Typo 😎  should be `s3_bucket_inventory.html`

Reply:
D'oh! Thanks for the fix.